workers/unity/Packages/com.improbable.gdk.deploymentlauncher/.DeploymentLauncher/Commands/Create.cs

```diff
@@ -0,0 +1,191 @@
+using System;
+using System.IO;
+using System.Net;
+using System.Security.Cryptography;
+using Google.Protobuf.WellKnownTypes;
+using Improbable.SpatialOS.Deployment.V1Alpha1;
+using Improbable.SpatialOS.PlayerAuth.V2Alpha1;
+using Improbable.SpatialOS.Snapshot.V1Alpha1;
+using Newtonsoft.Json.Linq;
+
+namespace Improbable.Gdk.DeploymentLauncher.Commands
+{
+    public static class Create
+    {
+        public static int CreateDeployment(Options.Create options)
+        {
+            return CreateDeploymentInternal(options, opts => File.ReadAllText(opts.LaunchJsonPath));
+        }
+
+        public static int CreateSimulatedPlayerDeployment(Options.CreateSimulated options)
+        {
+            return CreateDeploymentInternal(options, opts => ModifySimulatedPlayerLaunchJson(options));
+        }
+
+        private static int CreateDeploymentInternal<TOptions>(TOptions options, Func<TOptions, string> getLaunchConfigJson)
```

Comment:
Should make a standard to make named `delegate`s rather than using raw `Func<>` and `Action`? I think it may make things more readable in the long run.

Reply:
I think it probably depends on how exposed that `Func<>` and `Action` is. 

If its localized to a single file I think its probably _fine_ to keep them as raw `Func<>` and `Action` - if its an API (internal or external) then yes I would agree that we should use delegates